alloydb/notebooks/embeddings_batch_processing_e2e_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2022 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample creates a secure two-service application running on Cloud Run.
+# This test builds and deploys the two secure services
+# to test that they interact properly together.
+
+import asyncpg  # type: ignore
+import conftest as conftest  # python-docs-samples/alloydb/conftest.py
+import pytest
+import sqlalchemy
+from google.cloud.alloydb.connector import AsyncConnector, IPTypes
```

Comment:
To resolve linting, these two imports need to be swapped
```suggestion
from google.cloud.alloydb.connector import AsyncConnector, IPTypes
import sqlalchemy
```